### ISSUE TITLE
Public data sources redesign

### ISSUE THREAD

**Issue by orangemug** [maputnik/editor#643]
I think it'd be really good to redesign the 'public sources' section in data sources modal, linking to docs, pricing, stats, etc... for the different public data sources (Thunderforest, MapTiler, Ordnance Survey)

Current UI
<img width="615" alt="Screenshot 2020-04-12 at 10 38 51" src="https://user-images.githubusercontent.com/235915/79065653-fb8f2100-7ca9-11ea-98dd-2a3d2844aade.png">

What I personally think would be useful is

 - Links to pricing and usage tiers
 - Link to docs about the tilesets what features they provide etc...

Questions

 - Are there any other details we should provide in the UI?
 - Any other thoughts/ideas?

Pinging people at the various tileset providers @gravitystorm, @klokan, @snodnipper

**Comment by pathmapper** [comment]
> Are there any other details we should provide in the UI?

- Official feedback channel for reporting issues related to the data, see https://github.com/maputnik/editor/issues/664.

> Any other thoughts/ideas?

- We could also add some information to the public styles available in Maputnik ( 'gallery styles' section in open modal), e.g. the style repo.
![grafik](https://user-images.githubusercontent.com/20856381/80304234-f0151d00-87b4-11ea-9226-08b4d1668ebf.png)

**Comment by snodnipper** [comment]
The comments above sound great.  #ThoughtsOfOne:

* coverage - an indicator to suggest whether data coverage is global or not (e.g. globe icon if so).
* license - a subtle attribution value would be helpful.  Could get messy with maps mixing multiple data sources...maybe an ellipsis.
* data authority - perhaps a reference to OSM, OS etc. might be helpful.  Clearly OSM != OS and so one should expect differences in the source data.

👍

**Comment by pathmapper** [comment]
For styles we should add:
- Link to repository
- Link to license

ref: https://github.com/maputnik/editor/pull/704

**Comment by pathmapper** [comment]
For active and public sources we should add also 

- source type

**Comment by jo-chemla** [comment]
Just stumbled on this thread, some of the stale issues can probably be cleaned up now that Maputnik has moved under the MapLibre org?

**Comment by HarelM** [comment]
I don't this was resolved, but using AI to solve some of the issues here should be fairly easy.

**Comment by jo-chemla** [comment]
You're right the additional metadata requested on this thread - pricing, docs, coverage, license, repo etc - is not yet surfaced on each card of the public sources or gallery styles - my bad I skimmed through that thread too fast!
